app/Models/BoardPost.php

```diff
@@ -44,15 +44,15 @@ class BoardPost extends Model
 
     protected $guarded = [];
 
-    public function cat(){
+    public function cat() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->